organization/controllers.py

```diff
@@ -42,23 +43,55 @@
 TWITTER_ACCESS_TOKEN_SECRET = get_environment_variable("TWITTER_ACCESS_TOKEN_SECRET")
 
 
-def organization_retrieve_tweets(organization_we_vote_id, number_to_retrieve):
+def organization_retrieve_tweets(organization_we_vote_id, number_to_retrieve=5):
```

Comment:
Lets rename this organizations_retrieve_tweets_from_twitter so we don't confuse the function with other code that retrieve's tweets from our local cache.